src/bin/service_main.rs

```diff
@@ -143,146 +143,6 @@ async fn main() {
-}
-
-async fn build_package(
-    State(project_manager): State<Arc<Mutex<FlowProjectManager>>>,
-    Path(project_name): Path<String>,
-) -> Result<impl IntoResponse, (StatusCode, String)> {
-    let project = project_manager
-        .lock()
-        .unwrap()
-        .projects
-        .get(&project_name)
-        .cloned();
-
-    if let None = project {
-        let error_message = format!(
-            "The Project {} does not exist in the Project Manager",
-            project_name
-        );
-        eprintln!("{}", error_message);
-        return Err((StatusCode::INTERNAL_SERVER_ERROR, error_message.to_string()));
-    }
-
-    let project_path = PathBuf::from(format!("./flow-projects/{}", project_name));
-
-    if !project_path.exists() {
-        let error_message = "The specified project directory does not exist.";
-        eprintln!("{}", error_message);
-        return Err((StatusCode::INTERNAL_SERVER_ERROR, error_message.to_string()));
-    }
-
-    if let Err(err) = std::env::set_current_dir(&project_path) {
-        let error_message = format!("Failed to change the working directory: {}", err);
-        eprintln!("{} {}", error_message, err);
-        return Err((StatusCode::INTERNAL_SERVER_ERROR, error_message.to_string()));
-    }
-
-    let wasm_pack_build = Command::new("wasm-pack")
-        .args(&["build", "--target", "web"])
-        .output();
-
-    if let Err(err) = wasm_pack_build {
-        let error_message = format!("Couldn't build the project: {}", err);
-        eprintln!("{} {}", error_message, err);
-        return Err((StatusCode::INTERNAL_SERVER_ERROR, error_message.to_string()));
-    }
-
-    let output = wasm_pack_build.unwrap();
-
-    if !output.status.success() {
-        eprintln!("Failed to build the project for WebAssembly.");
-        if let Some(stderr) = String::from_utf8_lossy(&output.stderr)
-            .trim()
-            .to_string()
-            .split('\n')
-            .next()
-        {
-            eprintln!("Error: {}", stderr);
-        }
-    }
-
-    println!("Project built successfully!");
-    let target_dir = project_path.join("pkg");
-
-    if !target_dir.exists() {
-        let error_message = "The target dir of the generated WASM file cannot be found.";
-        eprintln!("{}", error_message);
-        return Err((StatusCode::INTERNAL_SERVER_ERROR, error_message.to_string()));
-    }
-
-    let file_name = format!("{}_bg.wasm", project_name);
-    let wasm_file_path = target_dir.join(file_name.clone());
-    let wasm_file = tokio::fs::File::open(wasm_file_path)
-        .await
-        .expect("Failed to open Wasm file");
-    let stream = ReaderStream::new(wasm_file);
-    let body = StreamBody::new(stream);
-    let headers = [
-        (header::CONTENT_TYPE, "application/wasm".to_string()),
-        (
-            header::CONTENT_DISPOSITION,
-            format!("attachment; filename=\"{:?}\"", file_name),
-        ),
-    ];
-
-    Ok((headers, body))
-}
```

Comment:
@FWuermse 

Das hab ich übernommen im compile and run in der Art. Allerdings hat man dann nicht mehr den Header und den Body der File. Ist das wichtig ? Dann nehme ich das wieder rein